packages/mongo/remote_collection_driver.js

```diff
@@ -75,4 +76,4 @@ MongoInternals.defaultRemoteCollectionDriver = _.once(function () {
   });
 
   return driver;
-});
+});
```

Comment:
Can you make sure not to remove those? Next person editing this file will add it again.